merge/action.yml

```diff
@@ -98,3 +104,9 @@ outputs:
         ...
       }
       ```
+  documented_spec_path:
```

Comment:
aside: I've always hated the "documented" lingo, we should gavel on what this should be called going forward, idk if there is an intentional decision behind this.

Reply:
no intentional decision, aiui. but "documented" is what we say in the studio and the docs